transcom/mymove
# Populates the expenses section of the Shipment Summary Worksheet

## Description

Populates the Claimable Expenses section on the Shipment Summary Worksheet

- NOTE: For the totals there is already text on the pdf template, which makes the totals hard to read. This will be fixed in a future story, which will incorporate multiple pdf updates.

## Setup

`make server_test`

1) Log into the office app and select a move with a PPM
2) Click the documents link and add an "Expense"
3) After you upload a document you'll see it's red on the right hand side. Select the document go to the details tab and set the document status to "OK".
4) Go back to the PPM on the move summary page and there should be some expenses now
5) Click Create Paperwork -> Click Download worksheet

## Code Review Verification Steps

## References

* [Pivotal story](https://www.pivotaltracker.com/n/projects/2181745/stories/163540220) 

## Screenshots
![image](https://user-images.githubusercontent.com/1036969/52508086-9d037a00-2bb0-11e9-80fc-c59f70831bb5.png)
